CONTRIBUTING.md

```diff
@@ -38,6 +38,17 @@ The process for reporting bugs and requesting smaller features is also outlined
 
 Pull Requests (PRs) should follow these guidelines as much as possible:
 
+### Creating a pull request from a fork
+Below is a general workflow to create a PR from a forked repository. By following these steps, we can avoid granting write permissions to the main repository.
+- [Fork the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)
+- Clone the forked repository and [Configuring a remote repository for a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork)
```

Comment:
nit: should Configuring a remote repository for a fork be a separate line item?

Reply:
done